### ISSUE TITLE
NameError: uninitialized constant Resque::Version

### ISSUE THREAD

**Issue by diei** [resque/resque-web#156]
After update from resque 2.1.0 to 2.2.0 I get following error when I load the resque-web website:
```
NameError: uninitialized constant Resque::Version
  from resque-web (0.0.12) app/views/layouts/resque_web/application.html.erb:73
```

The constant definition was changed in `resque/lib/resque/version.rb`:
From:
```
module Resque
  Version = VERSION = '2.0.0'
end
```
To:
```
module Resque
  VERSION = '2.0.0'
end
```

**Comment by jlkauffman92** [comment]
I'm getting this error as well.

**Comment by francescor** [comment]
we would like this fix too :)

**Comment by alexadia** [comment]
Likewise, would also like this fix.

**Comment by hackhowtofaq** [comment]
Anyone around to merge this, plz! ;)

**Comment by weshatheleopard** [comment]
Merge and release the new version please.

**Comment by wwwermishel** [comment]
Merge and release the new version please.

**Comment by bassboy2k** [comment]
New version please!

**Comment by garybunofsky** [comment]
New version plz

**Comment by kameshprasad** [comment]
Please merge and release new version

**Comment by jlkauffman92** [comment]
My work around for this is simply putting `Resque:: Version = 2` in my application.rb file.

**Comment by kameshprasad** [comment]
More accurate will be to have `Resque:: Version = Resque::VERSION` in `application.rb` or in `initilizers/resque.rb`

**Comment by trevorthornton** [comment]
```
Resque::Version = Resque::VERSION
```

FWIW - in my case (using Rails 7) that line had to go in `application.rb` – tried it in `initilizers/resque.rb` but it didn't work.

The fact that the fix for this is so simple and remains unmerged, and that the code hasn't been touched in over a year, makes me suspect that this gem may be endangered. What can we do to help?

**Comment by lancecarlson** [comment]
Yeah, this is such a simple fix. Why can't we get a new release for this?
